CONTRIBUTING.rst

```diff
@@ -0,0 +1,173 @@
+Contributing to Vulture
+=======================
+
+Setting up a virtualenv
+-----------------------
+Virtualenv is probably what you want to use during development.
+You can read more about
+it at the `virtualenv documentation <http://virtualenv.readthedocs.org>`_.
+
+First, we need to install virtualenv to the system. You may already 
+have this installed as ``virtualenv`` or ``pyvenv``. If you do not, 
+this can be done with ``pip`` easily:
+
+::
+
+    $ pip install virtualenv
+
+Once you have ``virtualenv`` installed, just fire up a shell and create
+your own environment:
+
+::
+
+    $ virtualenv ENV
+
+Where ``ENV`` is a directory to place the new virtual environment.
+
+Now, whenever you want to work on the project, you only have to activate 
+the corresponding environment.
+
+    On **Unix based** systems (OS X and Linux), this can be done with:
+
+    ::
+
+        $ source ENV/bin/activate
+
+    And on **Windows** this is done with:
+
+    ::
+
+        $ ENV\scripts\activate
+
+Note that if this works successfuly, your terminal prompt should look like:
+
+::
+
+    (ENV)$
+
+Finally, you should install vulture inside the activated virtualenv.
+
+In order to deactivate the ``virtualenv``, simply use
+
+::
+
+    (ENV)$ deactivate
+
+Installation
+------------
+
+There are two ways of installing vulture. By installing releases from PyPI, 
+or directly from source.
+
+PyPI Installation
+~~~~~~~~~~~~~~~~~
+
+::
+
+  $ pip install vulture
+
+Source installation
+~~~~~~~~~~~~~~~~~~~
+
+::
+
+  $ git clone https://github.com/jendrikseipp/vulture
+  $ cd vulture
+  $ pip install .
+
+Similarly, you can install your own fork of vulture.
```

Comment:
You need not install different forks. In the fork section (see comment above), we may add - You can add a remote pointing towards your fork by:
`git remote add origin https://github.com/USERNAME/vulture`